bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_10/A5_10_15.java

```diff
@@ -18,11 +18,45 @@
 /**
  *
  * @author Daniel Weber - Initial contribution
+ * 
+ *         From A5_10_15 up to A5_10_17 temperature is given as a 10Bit value (range: 1023..0).
+ *         Therefore higher values mean lower temperatures.
+ *         Temperature range -10..41.2.
```

Comment:
I can't make suggestions to existing code, but can you reformat this to:
```java
/**
 * From A5_10_15 up to A5_10_17 temperature is given as a 10Bit value (range: 1023..0).
 * Therefore higher values mean lower temperatures.
 * Temperature range -10..41.2.
 *
 * @author Daniel Weber - Initial contribution
 */
```

This will ensure it will be correctly shown in Javadoc and IDE's.

Reply:
🤣  I just though about where to put the comment, beneath or above the author tag. Thanks a lot for the hint 👍